app/workers/central_mail/submit_form4142_job.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+module CentralMail
+  class SubmitForm4142Job
+    include Sidekiq::Worker
+    include SentryLogging
+
+    FORM_ID = '21-4142'
+
+    FOREIGN_POSTALCODE = '00000'
+
+    # Sidekiq has built in exponential back-off functionality for retry's
+    # A max retry attempt of 13 will result in a run time of ~25 hours
+    RETRY = 13
+
+    sidekiq_options retry: RETRY
+
+    class CentralMailResponseError < Common::Exceptions::BackendServiceException
+    end
+
+    # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
+    sidekiq_retries_exhausted do |msg, _ex|
+      transaction_class.update_transaction(jid, :exhausted)
```

Comment:
If the retries are exhausted could we log the saved claim id (the `SavedClaim::DisabilityCompensation` record id not the claim id from EVSS that we send to the user). That will help with debugging as the 4142 data is stored there as well.

Reply:
I am unable to log the saved claim id as it would conflict 526 async transaction. The saved claim is for 526ez form and 4142 is embedded within 526 form. Not sure if there is a specific way you need this handled.

Reply:
I think we'll need to pass this in with the perform args. We're in the process of auditing the happy/unhappy logging for the flow so we could also add this later.